sqlalchemy-migrate.requirements

```diff
@@ -0,0 +1,8 @@
+decorator==4.0.10
+pbr==1.10.0
+pkg-resources==0.0.0
```

Comment:
I just ran `pip freeze > file` :) Maybe this one is not required.
Let me look at alembic, I may obsolete this PR.